### ISSUE TITLE
Identify a11y failures

### ISSUE THREAD

**Issue by eahlberg** [avanslaars/cypress-axe#16]
Is there another way to see the a11y failures than through the console? I'm finding it somewhat time consuming to identify the violating DOM node, it'd be neat if you could use regular Cypress functionality like highlighting the DOM node through the UI, etc.

**Comment by sebinsua** [comment]
We could add a configuration option to put details of the accessibility failures within the `Error#message`.

**Comment by eahlberg** [comment]
@sebinsua thanks for the response, and sorry for me being slow to respond.

Good to know that you're open towards adding it to the error message. Unfortunately I won't have time to dig in to it right now, but hopefully I can carve out some time and look at it in the future. I'm not sure how they do it in Cypress, but your link looks promising, thanks!

**Comment by avanslaars** [comment]
This PR identifies nodes in error and has been merged and published in the latest release (`0.6.0`)

https://github.com/avanslaars/cypress-axe/pull/25
